package.json

```diff
@@ -55,6 +58,7 @@
     "grunt-template-jasmine-requirejs": "0.2.3",
     "imports-loader": "0.6.5",
     "json-loader": "0.5.4",
+    "junit-report-builder": "1.1.1",
```

Comment:
We do need the shrinkwrap - @stsai-okta just went through this and has written documentation on it (can you link it here @stsai-okta?).

There are some issues with installsw, our version of npm, etc, but general gist is:
- Upgrade locally to 3.9.3
- Install a global version of install-with-shrinkwrap
- Modify local copy of install-with-shrinkwrap to comment out [this line](https://github.com/okta/npm/blob/master/install-with-shrinkwrap/lib/shrinkwrap.js#L21)
- Modify local copy to use higher version of npm [here](https://github.com/okta/npm/blob/master/install-with-shrinkwrap/package.json#L45)
- Use the global version to shrinkwrap